.travis.yml

```diff
@@ -17,7 +17,7 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   #Grab all dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION swig simplejson pandas opencv=3.1 coverage pip wheel ophyd==0.6.1 bluesky==0.9.0 pytest -c conda-forge -c lightsource2-tag
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION swig simplejson pydm psbeam pswalker==0.1.0 pcds-devices==0.1.0 pandas opencv=3.1 coverage pip wheel ophyd==0.6.1 bluesky==0.9.0 pytest -c pydm-dev  -c skywalker-tag -c conda-forge -c gsecars -c lightsource2-tag -c skywalker-dev
```

Comment:
We're moving the rest of snd to 1.0.0 bluesky and ophyd. Are your changes compatible with those?

Reply:
Ignore this, everything was compatible and I'm merging/rebasing/etc.